### PR TITLE
Import argentine superliga teams and season info

### PR DESCRIPTION
**Pull request by CaiqueCoelho**
This PR address issue #12 @richecr sorry about the delay with other issue hahaha

### CODE REVIEW

File: PySoccer/constants/UrlsLeagues.py [hunk 1]
```diff
@@ -12,7 +12,8 @@ def __init__(self):
             "italy_serie_a": self.get_url_leaderboard_italy,
             "portugal_nos_league": self.get_url_leaderboard_portugal,
             "germany_bundesliga": self.get_url_leaderboard_germany,
-            "france_ligue_one": self.get_url_leaderboard_france
+            "france_ligue_one": self.get_url_leaderboard_france,
+            "argentine_superliga": self.get_url_leaderboard_italy,
```
Comment (ghost) [inline PySoccer/constants/UrlsLeagues.py]: We needed to change this method to `get_url_leaderboard_argentine`.

**Review by ghost — CHANGES_REQUESTED** [review]

File: PySoccer/constants/UrlsLeagues.py (hunk 1 above)
Comment (CaiqueCoelho) [reply PySoccer/constants/UrlsLeagues.py]: My bad resolved :D

**Comment by CaiqueCoelho** [comment]
@richecr before you close, can you please put hacktoberfet tag in the issue so that this pr can count in my contributions, please?

**Review by ghost — APPROVED** [review]
Very good, thanks for the contribution!

**Pull request merged**